doc/src/tutorial/compile_cmake.dox

```diff
@@ -1,57 +1,249 @@
-/** @page compile_cmake PortAudio on Windows, OS X or Linux via. CMake
+/** @page compile_cmake Building PortAudio with CMake
 @ingroup tutorial
 
-@section cmake_building Building PortAudio stand-alone on Windows, OS X or Linux
+As an alternative to autotools, PortAudio offers a way of building via CMake. CMake works by generating files for other build systems, including Visual Studio solutions, Makefiles, and many others others.
 
-CMake can be used to generate Visual Studio solutions on Windows, Makefiles (on Linux and OS X) and build metadata for other build systems for PortAudio. You should obtain a recent version of CMake from [http://www.cmake.org] if you do not have one already. If you are unfamiliar with CMake, this section will provide some information on using CMake to build PortAudio.
+If you are familiar with building CMake projects, PortAudio follows the standard steps of building a CMake project. Otherwise, continue reading on.
 
-On Linux, CMake serves a very similar purpose to an autotools "configure" script - except it can generate build metadata apart from Makefiles. The equivalent of the following on POSIX'y systems:
+@section compile_cmake_general General Steps
 
-    build_path> {portaudio path}/configure --prefix=/install_location
-    build_path> make
-    build_path> make install
+The build process for a CMake project is pretty much the same across all platforms. To get you up and running as quickly as possible, we'll begin with general instructions that explain how to build PortAudio _on all platforms_. If you need further assistance, later sections explain how to build for your specific platform. (See @ref compile_cmake_win for Windows and @ref compile_cmake_others for macOS, Linux, and others, respectively).
 
-Would be:
+@subsection compile_cmake_general_clone Step 1: Get the PortAudio Source Code
 
-    build_path> cmake {portaudio path} -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=/install_location
-    build_path> make
-    build_path> make install
+Before you proceed with building PortAudio, you must ensure that you have the PortAudio source code. If you don't, you will first need to clone it from PortAudio's GitHub repository using Git. If you do not have Git installed, install it either by downloading an installer from https://git-scm.org (Windows and macOS) or using your system's package manager (Linux, FreeBSD, etc). Then, open a terminal, command prompt window, or PowerShell window, in a suitable directory (e.g., your documents folder) and run the following command:
```

Comment:
This is OK, but note that git is only one way to download PortAudio source code.